### PR TITLE
Separate config files

### PR DESCRIPTION
**Pull request by lornat75**

### CODE REVIEW

File: app/iCubSim/test_ft_right_leg.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name "Test FT sensors Right Leg"
+portname /${robotname}/right_leg/analog:o
```
Comment (traversaro) [inline app/iCubSim/test_ft_right_leg.ini]: The FT sensors are not present in the iCubSim ODE simulator, however are present in the real robot, so I think we should move this configuration file in the iCubTest config.

File: app/iCubTest/test.ini [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+[ENVIRONMENT]
+robotname icub
+
+[TESTS]
+TestCamera right_camera.ini
+TestCamera left_camera.ini
+TestMotors test_left_arm.ini
+TestMotors test_right_arm.ini
+
```
Comment (traversaro) [inline app/iCubTest/test.ini]: Some sensors are available on only some iCubs. Should do we move at least the test.ini file in robots specific directories?

Comment (lornat75) [inline app/iCubTest/test.ini]: Right. Probably!

File: app/iCubSim/test_ft_right_leg.ini (hunk 1 above)
Comment (lornat75) [inline app/iCubSim/test_ft_right_leg.ini]: correct, done.

**Pull request merged**